torchvision/csrc/io/image/cpu/decode_jpeg.cpp

```diff
@@ -102,20 +157,29 @@ torch::Tensor decode_jpeg(const torch::Tensor& data, ImageReadMode mode) {
   jpeg_read_header(&cinfo, TRUE);
 
   int channels = cinfo.num_components;
+  bool cmyk_to_rgb = false;
```

Comment:
Since this is also true when we convert to grey:

```suggestion
  bool cmyk_to_other = false;
```

Reply:
well, "other" is not a appropriate name as it is too inaccurate. Here `cmyk_to_rgb` still make sense as even when we convert cmyk to gray, we still pass by rgb.

Reply:
`cmyk_to_rgb_or_gray` then? `cmyk_to_rgb` is not a good name, sorry